docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -630,6 +630,12 @@ Set this flag to `true` to enable audit logging for the universe.
 
 For details, see [Audit logging for the YCQL API](../../../secure/audit-logging/audit-logging-ycql).
 
+##### --ycql-allow-non-authenticated-password-reset
```

Comment:
Please update it to `--ycql_allow_non_authenticated_password_reset` i.e. use underscore instead of dash.